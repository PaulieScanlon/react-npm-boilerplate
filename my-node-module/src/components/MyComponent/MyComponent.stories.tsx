import React from 'react'

import { MyComponent } from './MyComponent'

export default {
  title: 'MyComponent',
  parameters: {
    component: MyComponent,
    componentSubtitle: 'MyComponent is ...',
  },
}

export const usage = () => <MyComponent />
