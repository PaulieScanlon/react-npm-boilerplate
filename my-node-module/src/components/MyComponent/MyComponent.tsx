import React from 'react'
import { Box } from 'theme-ui'

export const MyComponent = () => {
  return <Box>MyComponent</Box>
}
