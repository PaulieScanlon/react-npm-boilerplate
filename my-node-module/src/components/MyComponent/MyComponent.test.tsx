import { render } from '@testing-library/react'

import { usage } from './MyComponent.stories'

describe('<MyComponent />', () => {
  test('renders the component', () => {
    const { asFragment } = render(usage())
    expect(asFragment()).toMatchSnapshot()
  })
})
