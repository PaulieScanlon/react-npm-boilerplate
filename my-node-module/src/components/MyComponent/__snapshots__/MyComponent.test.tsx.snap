// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyComponent /> renders the component 1`] = `
<DocumentFragment>
  <div
    class="css-vurnku"
  >
    MyComponent
  </div>
</DocumentFragment>
`;
